include/map_object.h

```diff
@@ -349,15 +335,15 @@ void sub_0205F490(LocalMapObject* object, LocalMapObject_UnkCallback callback);
 void sub_0205F498(LocalMapObject* object);
```

Comment:
Do we want to rename LocalMapObject -> MapObject? what does local convey here?

Reply:
ehh, it's local to the current context, we *can* make it MapObject but this is out of scope of this PR